lib/prelude/sprite/system.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:backbone/filter.dart';
+import 'package:backbone/prelude/sprite/trait.dart';
+import 'package:backbone/prelude/transform.dart';
+import 'package:backbone/realm.dart';
+import 'package:flame/components.dart';
+
+/// Render Sprite- and SpriteAnimationComponent
+void spriteSystem(Realm realm) {
+  final realmQuery = realm.query(Has([SpriteTrait]), onlyLoaded: true);
+
+  for (final node in realmQuery) {
+    final spriteTrait = node.get<SpriteTrait>();
+    if (spriteTrait.dirty) {
```

Comment:
Just a passing thought, but can we try to implement it without a `dirty` flag, like after the changes we did for the input system and transform?

Reply:
Might really help us with not having any delay frames when changing stuff, might also make out code simpler.

Reply:
We need it because the traits properties can be set at any time. The actual target component might not be there yet.